guides/source/active_storage_overview.md

```diff
@@ -687,6 +687,38 @@ previewable files. You can also call these methods directly.
 [`representable?`]: https://api.rubyonrails.org/classes/ActiveStorage/Blob/Representable.html#method-i-representable-3F
 [`representation`]: https://api.rubyonrails.org/classes/ActiveStorage/Blob/Representable.html#method-i-representation
 
+### Lazy vs Immediate Loading
```

Comment:
I think loading is a bit ambiguous here.
Maybe:
```suggestion
### Lazy vs Immediate Processing
```

Reply:
I think "lazy loading" is a much more commonly used term to describe the pattern.

![image](https://user-images.githubusercontent.com/509837/122248215-8ca0ef80-ce8d-11eb-973a-15ac4396b339.png)
![image](https://user-images.githubusercontent.com/509837/122248264-9591c100-ce8d-11eb-9370-5d1cba244e7d.png)

Reply:
Ok, but in this case it's mostly about deferring the processing to another request right? At least the first time, when blobs haven't been processed...